### PR TITLE
Add fire event. Closes #35

### LINKED ISSUE #35: Fire
What if huts needed to be rebuilt like traps? It would have to be rare, huts are a huge wood drain.

### PR DESCRIPTION
**Pull request by anubisthejackle**

### CODE REVIEW

**Comment by anubisthejackle** [comment]
Updates commited.

**Comment by skj3gg** [comment]
If you know how, please add these to the `.pot` translation file as well.

**Comment by anubisthejackle** [comment]
I see a basic theme with the .pot file. Question I have is, there are lines with specific line numbers associated with them. Is this important, or is simply a reference, because my adding the Fire event to the top of the room.js is going to move those line numbers around.

**Comment by anubisthejackle** [comment]
I'll hold off on the .pot file until a decision is made on PR #223

**Comment by skj3gg** [comment]
You'll want to use Poedit or an equivalent and it should take care of that for you.

**Comment by anubisthejackle** [comment]
I've downloaded poedit and I'm able to load the .pot file into it, but not sure what exactly I'm supposed to do with it?

**Comment by skj3gg** [comment]
There... I'm at a loss for you.

[maybe](http://stackoverflow.com/questions/4832688/xgettext-extract-translatable-strings-and-update-pot)?

**Comment by anubisthejackle** [comment]
I "think" I figured it out.

**Comment by Continuities** [comment]
Great event, but I think it belongs outside and not in the room. Generally, "bad stuff" only happens when the player is lingering in the forest/village, and the room is more about resource trading and gambling. I should probably put that in the style guide...

File: script/events/room.js [hunk 1]
```diff
@@ -49,7 +49,37 @@ Events.Room = [
...
+	}, 
+	{
+		title: _('Fire'),
+		isAvailable: function() { 
+			return $SM.get('game.buildings["hut"]', true) > 0 && $SM.get('game.population', true) > 5;
+		},
+		scenes: {
+			'start': {
+				text: [
+					_('A fire rampages through one of your huts, destroying it.'),
```
Comment (Continuities) [inline script/events/room.js]: Would be better as "a fire rampages through one of the huts, destroying it." ADR doesn't use personal pronouns or capital letters.

File: script/events/room.js [hunk 2]
```diff
@@ -49,7 +49,37 @@ Events.Room = [
...
+					_('Saddly, all residents in the hut perished in the fire.')
```
Comment (Continuities) [inline script/events/room.js]: I would probably take "sadly" out. ADR is best when it's stark and unforgiving. It's not sad, it's life.

**Comment by skj3gg** [comment]
Looks great and is committed, THANK!

**Comment by anubisthejackle** [comment]
mmmmm... cake....

Drool face

**Comment by daveschaefer** [comment]
@Continuities Style guide updated with these notes. Great stuff. Shout if something is missing.

**Comment by Continuities** [comment]
Badass. You folks rock.

**Pull request closed without merging**